src/l10n.py

```diff
@@ -109,7 +115,9 @@ def formatTimestamp(timestamp = None, as_unicode = True):
         return unicode(timestring, encoding)
     return timestring
 
+
 def getTranslationLanguage():
+    """Return the user's language choice"""
     userlocale = None
     if BMConfigParser().has_option('bitmessagesettings', 'userlocale'):
         userlocale = BMConfigParser().get('bitmessagesettings', 'userlocale')
```

Comment:
This may be simplified:
```python
userlocale = BMConfigParser().safeGet(
    'bitmessagesettings', 'userlocale', 'system')
return userlocale if userlocale and userlocale != 'system' else language
```

Reply:
Thanks